pom.xml

```diff
@@ -879,6 +880,26 @@
                 <version>${antlr4.version}</version>
             </dependency>
 
+            <!-- https://mvnrepository.com/artifact/org.jsoup/jsoup -->
```

Comment:
Do we need both jsoup and owasp here?

Reply:
Oh mistake from my part, need to remove those.